analyzere/__init__.py

```diff
@@ -2,6 +2,7 @@
 password = ''
 base_url = 'http://localhost:8000/'
 upload_poll_interval = 0.1
+upload_chunk_size = 16 * (1 << 20)
```

Comment:
IMO it's a little more readable as `16 * 1024 * 1024` but that's probably just me.